[CI] EsqlClientYamlIT test {p0=esql/26_aggs_bucket/friendlier BUCKET interval hourly: #110916} failing

**Build Scans:**
- [elasticsearch-intake #8929 / 8.15.1_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ccntcazjedxhm)
- [elasticsearch-intake #8927 / 8.15.1_bwc-snapshots](https://gradle-enterprise.elastic.co/s/7pkml7meflile)
- [elasticsearch-pull-request #28460 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/37klh2l7eguv4)
- [elasticsearch-pull-request #28460 / 8.15.1_bwc](https://gradle-enterprise.elastic.co/s/7gqfv7gm3ttz2)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:mixed-cluster:v8.15.1#yamlRestTest" -Dtests.class="org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT" -Dtests.method="test {p0=esql/26_aggs_bucket/friendlier BUCKET interval hourly: #110916}" -Dtests.seed=4C7FFE4BAC04942A -Dtests.bwc=true -Dtests.locale=id -Dtests.timezone=America/Metlakatla -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bp0%3Desql%2F26_aggs_bucket%2Ffriendlier%20BUCKET%20interval%20hourly%3A%20%23110916%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [esql/26_aggs_bucket:39]: field [values] doesn't have length [4]
Expected: <4>
     but: was <2>
```

**Issue Reasons:**
- [main] 4 failures in test test {p0=esql/26_aggs_bucket/friendlier BUCKET interval hourly: #110916} (20.0% fail rate in 20 executions)
- [main] 2 failures in step 8.15.1_bwc-snapshots (33.3% fail rate in 6 executions)
- [main] 2 failures in pipeline elasticsearch-intake (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -122,12 +122,6 @@ tests:
 - class: org.elasticsearch.xpack.restart.CoreFullClusterRestartIT
   method: testSnapshotRestore {cluster=UPGRADED}
   issue: https://github.com/elastic/elasticsearch/issues/111799
-- class: org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT
-  method: "test {p0=esql/26_aggs_bucket/friendlier BUCKET interval hourly: #110916}"
-  issue: https://github.com/elastic/elasticsearch/issues/111901
-- class: org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT
-  method: "test {p0=esql/26_aggs_bucket/friendlier BUCKET interval: monthly #110916}"
-  issue: https://github.com/elastic/elasticsearch/issues/111902
 - class: org.elasticsearch.xpack.esql.qa.mixed.FieldExtractorIT
   method: testScaledFloat
   issue: https://github.com/elastic/elasticsearch/issues/112003

